frontend/package.json

```diff
@@ -95,7 +95,7 @@
     "build": "NODE_OPTIONS=\"--max_old_space_size=2048\" next build",
     "start": "next start",
     "develop": "next dev",
-    "format": "npx prettier --write .",
+    "format": "concurrently \"npx prettier --write .\" \"node_modules/.bin/next lint --fix\" \"node_modules/.bin/stylelint --fix '**/*.{js,ts,tsx,css}'\"",
```

Comment:
Ooh thanks for adding `concurrently` here! I think this is a great speedup with probably a small chance of different commands competing to write to the same file? Don't think it'll happen any time soon, but maybe good for us to keep in mind 😆 !

Reply:
hmmm should we run the commands sequentially then? I think the small speed up might not be worth any collisions 😅